samples/bot-sequential-flow-adaptive-cards/csharp/SequentialUserSpecificFlow/Models/InitialSequentialCard.cs

```diff
@@ -1,25 +1,52 @@
-﻿namespace SequentialUserSpecificFlow.Models
+﻿// <copyright file="InitialSequentialCard.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using System;
+
+namespace SequentialUserSpecificFlow.Models
 {
+    /// <summary>
+    /// Initial sequential card model class.
+    /// </summary>
     public class InitialSequentialCard
     {
         public Action action { get; set; }
-        public string trigger { get; set; }
     }
+
+    /// <summary>
+    /// Action model class.
+    /// </summary>
     public class Action
     {
         public string type { get; set; }
+
         public string title { get; set; }
+
         public Data data { get; set; }
+
         public string verb { get; set; }
     }
+
+    /// <summary>
+    /// Data model class.
+    /// </summary>
     public class Data
```

Comment:
What data is it holding?
Either Rename the class or add proper summary for class or properties.